package.json

```diff
@@ -1,9 +1,17 @@
 {
   "name": "probot-stale",
   "version": "1.1.0",
-  "description": "a GitHub App that closes abandoned issues after a period of inactivity",
+  "description": "A GitHub App built with Probot that closes abandoned Issues and Pull Requests after a period of inactivity.",
   "author": "Brandon Keepers",
   "license": "ISC",
+  "homepage": "https://probot.github.io/apps/stale/",
+  "bugs": "https://github.com/probot/stale/issues",
+  "keywords": [
+    "probot",
+    "github",
+    "probot-plugin",
```

Comment:
Again, we deprecated the probot-plugin terminology and usage, so I'd prefer not to include it here.

Reply:
Cool. Again; I would remove it from the GitHub keywords, then. :)